packages/plugin-eslint/src/lib/meta/rules.ts

```diff
@@ -27,7 +28,7 @@ export async function listRules(
     {
       messages: rulesIds.map(ruleId => ({ ruleId })),
       suppressedMessages: [] as Linter.SuppressedLintMessage[],
-    } as ESLint.LintResult,
+    } as unknown as ESLint.LintResult,
```

Comment:
Really not a fan of `as unknown as T`. It completely ignores any compatibiliy, while for `as T` TS at least checks it's "similar". Maybe adding `as Linter.LintMessage[]` to `messages` would work?

Reply:
No, it does not work. The ESLint problem is that you are using `as` as opposed to `satisfies` 🙄 . So I disabled that.

Reply:
OK, then you picked the best solution :+1: